coingecko-client/src/lib.rs

```diff
@@ -0,0 +1,129 @@
+use anyhow::Context;
```

Comment:
I took this package from [here](https://github.com/p2p-org/token-service/blob/main/price-service/src/sources/coingecko/client.rs)

Reply:
[This](https://github.com/p2p-org/token-service/blob/main/price-service/src/sources/coingecko/client.rs) part i'll move in next pr